plutus-tutorial/tutorial/Tutorial/03-wallet-api.md

```diff
@@ -133,6 +126,7 @@ Before we check whether `act` is permitted, we define a number of intermediate v
                   (&&) :: Bool -> Bool -> Bool
                   (&&) = $$(P.and)
 
+                  
                   signedBy :: PubKey -> Signature -> Bool
```

Comment:
It would be nice if we fixed up the signature stuff before the course, but that may be ambitious.